docs/modules/ROOT/pages/installing.adoc

```diff
@@ -17,12 +17,12 @@ You will also find links to source code archives and older versions on the downl
 == System Requirements
 
 * Windows 7+
-* Mac OS X 10.7+ (*64-bit only*)
-* CentOS 6 & 7 (64-bit only)
-* Debian 8.0 & 9.0
-* Fedora 25 & 26 & 27
-* Ubuntu 16.04 & 17.04 & 17.10
-* openSUSE Leap 42.2 & 42.3
+* Mac OS X 10.10+ (*64-bit only*)
+* CentOS 7 (64-bit only) & 8
```

Comment:
With the 2.6.1, C6 is still supported?

Reply:
Yes, support dropped after the release https://central.owncloud.org/t/desktop-client-supported-platforms-updated/24165

Reply:
No, not dropped after release of the 2.6.1. The release **after** the 2.6.1 won't support it. As long as the 2.6.1 is the latest stable, official release, C6 should be supported. The day we have a 2.6.2 or a 2.7.0, C6 is no longer supported.